### DISCUSSION TITLE
Modify to work in an non-internet connected environment, loading locally (on same network) hosted http pages. FullPageOS is staying on the splash screen

### DISCUSSION THREAD

**Discussion by genestealer** [guysoft/FullPageOS#607] [Q&A] [answered]
#### What were you doing?

Modify to work in an non-internet connected environment.
I use this to load locally (on same network) hosted http pages.
FullPageOS is staying on the splash screen

#### What did you expect to happen?
Local site to open

#### What happened instead?
FullPageOS is staying on the splash screen

#### Was there an error message displayed? What did it say?
None

#### Version of FullPageOS?

0.14.0

#### Screenshot(s) showing the problem:

[If applicable. Always include if unsure or reporting UI issues.]

**Comment by guysoft** [comment, marked as answer]
Check if the page you are trying to access is indeed accessible.

You can disable the network check and then it would work. Use ctrl +t to open a new tab once the browser loaded.

Put in a file called /boot/firmware/check_for_httpd the word disabled.
Code that checks that is here:
https://github.com/guysoft/FullPageOS/blob/devel/src/modules/fullpageos/filesystem/opt/custompios/scripts/run_onepageos#L9

Also converting to a discussion sine this is not a bug.

**Reply by africleap** [reply]
Using version 0.14.0 and encountered the same issue when iI change the hostname form fullpageos to something else. Basically chromium locks the profile until it is resolved.

**Reply by guysoft** [reply]
What do you mean by "locks?" What error do you get?
